source/simulator/nullspace.cc

```diff
@@ -221,13 +221,6 @@ namespace aspect
 
           }
 
-#ifdef DEBUG
-      Assert(n_left_to_find == 0, ExcMessage("Error, couldn't find a velocity DoF to constrain."));
-
-      for (unsigned int d=0; d<dim; ++d)
-        Assert(vel_idx[d] != numbers::invalid_dof_index,
-               ExcMessage("Error, couldn't find a velocity DoF to constrain."));
-#endif
```

Comment:
The code is supposed to check for something that shouldn't happen. If you remove it, is the surrounding code able to handle the situation that isn't supposed to happen, or will we now just silently get wrong results where we previously got an error?